duckduckgo/iOS
# NetP Geoswitching Design Review feedback

Task/Issue URL: https://app.asana.com/0/0/1206040993247659/f

**Description**:

Just requires an addition of a period at the end of a footer string + an updated design for the Preferred Location item on Settings as per designs (see task).

**Steps to test this PR**:
1. Make sure you’re an internal user
2. Go to Settings -> Network Protection -> VPN Settings
3. **Check the preferred location item matches designs**
4. Go to Preferred location
5. **Check the footer has a period at the end** 

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
